hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/codec/NetworkingStakingTranslator.java

```diff
@@ -23,6 +23,10 @@
 
 public final class NetworkingStakingTranslator {
 
+    private NetworkingStakingTranslator() {
+        throw new IllegalStateException("Utility class");
```

Comment:
All these private constructors throw `UnsupportedOperationException` in other classes. To keep it consistent, can change it to throw UnsupportedOperationException